doc/api/dns.md

```diff
@@ -417,6 +418,51 @@ is a two-dimensional array of the text records available for `hostname` (e.g.,
 one record. Depending on the use case, these could be either joined together or
 treated separately.
 
+## dns.resolveAny(hostname, callback)
+
+- `hostname` {string}
+- `callback` {Function}
+  - `err` {Error}
+  - `ret` {Object[][]}
+
+Uses the DNS protocol to resolve any queries (`ANY` records) for the `hostname`.
```

Comment:
"to resolve all records (also known as `ANY` or `*` query)"

(This is taken out of the rfc - `255 A request for all records`)